### PR TITLE
[docker] ```AUTH_KEY``` arguments removed in ```examples/docker``` folder

### PR DESCRIPTION
**Pull request by Av1nag**
As of my knowledge, in the latest update there's no longer required any ```AUTH_KEY``` for installing. But the Docker file haven't updated with respect to the latest release. By address this issue, I've modified the all the three files in ```examples/docker```. Now can be build docker image without any Hassel.

### CODE REVIEW

**Review by JoeLoser — APPROVED** [review]

File: examples/docker/Dockerfile.mojosdk [hunk 1]
```diff
@@ -51,11 +51,8 @@ RUN pip install \
         ipywidgets
 
 # A random default token
-ARG AUTH_KEY=5ca1ab1e
-ENV AUTH_KEY=$AUTH_KEY
```
[the remark this answers is not in the document]
Comment (Av1nag) [reply examples/docker/Dockerfile.mojosdk:55]: Sure,  @JoeLoser

[the remark this answers is not in the document]
Comment (Av1nag) [reply examples/docker/Dockerfile.mojosdk:55]: I'm less aware of some git commands. So confused a bit. Hope you can understand @JoeLoser and @jackos

**Comment by JoeLoser** [comment]
!sync

**Comment by Av1nag** [comment]
hi @JoeLoser the version in ``` stdlib/COMPATIBLE_COMPILER_VERSION ``` was changed by @modularbot . Could you check it once. If required any changes from my side I'll do the needful if I can

**Pull request closed without merging**